tests/Functional/Responses/Expected/Api/WishlistTest/1.12/test_user_can_add_product_to_wishlist.json

```diff
@@ -0,0 +1,29 @@
+{
+  "@context":"\/api\/v2\/contexts\/Wishlist",
```

Comment:
Why some of the resource files are only for 1.12? Are the endpoints removed in 1.13?

Reply:
There is a directory for 1.13 responses. It's split because of different responses - for example in 1.13 shop user got id in serialization so we got additional id in response.